package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-dotnet-pack",
-  "version": "1.0.12",
+  "version": "1.0.13",
```

Comment:
@colombod Do we also need to update the minimum required vscode version below, just like we did last time when we moved to .NET 7 SDK as the jupyter extension needed vscode 1.74 or greater:

```json
"engines": {
    "vscode": "^1.74.0"
  },
```

Reply:
@arkalyanms I do see the CDK requires vscode version 1.80 or greater, so probably it makes sense to update the require vscode for this extension pack to the same?

Reply:
i would fix the minimum version to 1.83 as this is the version that integrates some api change in vscode that polyglot notebook relies upon.